edx_name_affirmation/tests/test_models.py

```diff
@@ -44,4 +49,37 @@ def test_histories(self):
 
         second_history_record = verified_name_history[1]
         assert second_history_record.status == VerifiedNameStatus.APPROVED
-        assert second_history_record.verification_attempt_id == idv_attempt_id
+        assert second_history_record.verification_attempt_id == self.idv_attempt_id
+
+    @patch('edx_name_affirmation.models.SoftwareSecurePhotoVerification')
+    def test_verification_status(self, sspv_mock):
+        """
+        Test the model history is recording records as expected
+        """
+
+        idv_attempt_id_notfound_status = None
+
+        sspv_mock.objects.get = self._mocked_model_get
+
+        self.verified_name.verification_attempt_id = self.idv_attempt_id_notfound
+        assert self.verified_name.verification_attempt_status is idv_attempt_id_notfound_status
+
+        self.verified_name.verification_attempt_id = self.idv_attempt_id
+        assert self.verified_name.verification_attempt_status is self.idv_attempt_status
+
+    # Helper methods
+
+    def _obj(self, dictionary):
+        "Helper method to turn a dict into an object. Used to mock below."
+
+        return type('obj', (object,), dictionary)
+
+    def _mocked_model_get(self, id):  # pylint: disable=redefined-builtin
+        "Helper method to mock the behavior of SoftwareSecurePhotoVerification model. Used to mock below."
+        if id == self.idv_attempt_id_notfound:
+            raise ObjectDoesNotExist
+
+        if id == self.idv_attempt_id:
+            return self._obj({'status': self.idv_attempt_status})
+
+        return self._obj({'status': None})
```

Comment:
I think we can omit the `Missing coverage` warning since this is just a mock for testing. I'm not sure why is this a thing on a test file.

Reply:
I'm not sure. I think it's fine to ignore!